From 657eec34d56f005f9c5e8adacb96d5c3d16c0d55 Mon Sep 17 00:00:00 2001
From: Charles Short <charles.short@windriver.com>
Date: Wed, 13 Apr 2022 09:08:27 -0400
Subject: [PATCH] Change group,passwd

Porting this patch from 'setup' package of CentOS. Removing the
change of uidgid, because no such file in base-passwd, and Debian
system. The file is installed into /usr/share/doc/setup-2.8.71/uidgid
in CentOS, which is just a doc file, so ignore it.

Signed-off-by: Andy Ning <andy.ning@windriver.com>
Signed-off-by: Al Bailey <Al.Bailey@windriver.com>
Signed-off-by: Yue Tao <yue.tao@windriver.com>
Signed-off-by: Charles Short <charles.short@windriver.com>
Signed-off-by: Dan Voiculeasa <dan.voiculeasa@windriver.com>
---
 group.master  | 3 +++
 passwd.master | 3 +++
 2 files changed, 6 insertions(+)

diff --git a/group.master b/group.master
index ad1dd2d..0037352 100644
--- a/group.master
+++ b/group.master
@@ -37,3 +37,6 @@ staff:*:50:
 games:*:60:
 users:*:100:
 nogroup:*:65534:
+snmpd:x:169:snmpd,fm
+fm:x:195:fm
+www:x:1877:www
diff --git a/passwd.master b/passwd.master
index f1e69a4..31bdc28 100644
--- a/passwd.master
+++ b/passwd.master
@@ -16,3 +16,6 @@ list:*:38:38:Mailing List Manager:/var/list:/usr/sbin/nologin
 irc:*:39:39:ircd:/run/ircd:/usr/sbin/nologin
 gnats:*:41:41:Gnats Bug-Reporting System (admin):/var/lib/gnats:/usr/sbin/nologin
 nobody:*:65534:65534:nobody:/nonexistent:/usr/sbin/nologin
+sysinv:x:168:168:sysinv Daemons:/var/lib/sysinv:/sbin/nologin
+snmpd:x:169:169:net-snmp:/usr/share/snmp:/sbin/nologin
+fm:x:195:195:fm-mgr:/var/lib/fm:/sbin/nologin
-- 
2.25.1

